setup_codecov.sh

```diff
@@ -4,11 +4,34 @@ set -euo pipefail
 
 CODECOV_VERSION=0.6.1
 
+# Check if the architecture was provided
+if [ -z "$1" ]; then
+  echo "Usage: $0 <architecture>"
+  echo "Example: $0 arm64"
+  exit 1
+fi
+
+ARCH=$1
+
+case $ARCH in
```

Comment:
Just an idea but we already have `DD_TARGET_ARCH` in every docker file (e.g. [here](https://github.com/DataDog/datadog-agent-buildimages/blob/f6a5fc0c98e06790844e0d2e0c9c64caceefa7bb/deb-arm/Dockerfile#L26))

Would it make sense to use it as a parameter to have the exact same thing in all the dockerfiles and handle the possible values in the script?

Reply:
(That's also an env variable)

Reply:
Good point thanks ! Fixed in [7823589](https://github.com/DataDog/datadog-agent-buildimages/pull/589/commits/7823589373f3deba6678c3d7c74168a3d1fd1220).